src/proxy-ui-api/frontend/src/views/Service/Parameters/ServiceParameters.vue

```diff
@@ -0,0 +1,488 @@
+                                        color="primary"
+                                        class="xrd-small-button"
+                                        @click="removeSubject(subject)"
+                                        data-test="remove-subject"
+                                >{{$t('action.remove')}}
+                                </v-btn>
+                            </div>
+                        </td>
+                    </tr>
+                </template>
+            </table>
+
+            <div class="footer-buttons-wrap">
+                <large-button @click="close()" data-test="close">{{$t('action.close')}}</large-button>
+            </div>
+        </v-card>
+
+        <!-- Confirm dialog remove Access Right subject -->
+        <confirmDialog
+                :dialog="confirmMember"
+                title="localGroup.removeTitle"
+                text="localGroup.removeText"
+                @cancel="confirmMember = false"
+                @accept="doRemoveSubject()"
+        />
+
+        <!-- Confirm dialog remove all Access Right subjects -->
+        <confirmDialog
+                :dialog="confirmAllSubjects"
+                title="localGroup.removeAllTitle"
+                text="localGroup.removeAllText"
+                @cancel="confirmAllSubjects = false"
+                @accept="doRemoveAllSubjects()"
+        />
+
+        <!-- Add access right subjects dialog -->
+        <accessRightsDialog
+                :dialog="addSubjectsDialogVisible"
+                :filtered="accessRightsSubjects"
+                :clientId="clientId"
+                title="access.addSubjectsTitle"
+                @cancel="closeAccessRightsDialog"
+                @subjectsAdded="doAddSubjects"
+        />
+    </div>
+</template>
+
+
+<script lang="ts">
+import Vue from 'vue';
+import * as api from '@/util/api';
+import AccessRightsDialog from '../AccessRightsDialog.vue';
+import ConfirmDialog from '@/components/ui/ConfirmDialog.vue';
+import HelpIcon from '@/components/ui/HelpIcon.vue';
+import LargeButton from '@/components/ui/LargeButton.vue';
+import {ServiceClient} from '@/types';
+import {ValidationObserver, ValidationProvider} from 'vee-validate';
+import {mapGetters} from 'vuex';
+import {RouteName} from '@/global';
+
+type NullableSubject = undefined | ServiceClient;
+
+export default Vue.extend({
+  components: {
+    AccessRightsDialog,
+    ConfirmDialog,
+    HelpIcon,
+    LargeButton,
+    ValidationProvider,
+    ValidationObserver,
+  },
+  props: {
+    serviceId: {
+      type: String,
+      required: true,
+    },
+    clientId: {
+      type: String,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      touched: false,
+      confirmGroup: false,
+      confirmMember: false,
+      confirmAllSubjects: false,
+      selectedMember: undefined as NullableSubject,
+      description: undefined,
+      url: '',
+      addSubjectsDialogVisible: false,
+      timeout: 23,
```

Comment:
Timeout is hardcoded  to 23 ?

Reply:
This haven't been changed. Don't actually know what is this timeout variable for but it hasn't changed in this task (just moved to another file)